dns/push.sh

```diff
@@ -15,14 +15,31 @@
 # limitations under the License.
 # This runs as you.  It assumes you have built an image named ${USER}/octodns.
 
+CANARY_ZONES=(
```

Comment:
should we auto derive these from PROD_ZONES? they are all `"canary.${prod_zone}"` right?

Reply:
Done